foxy/distribution.yaml

```diff
@@ -2767,13 +2767,13 @@ repositories:
       - swri_yaml_util
       tags:
         release: release/foxy/{package}/{version}
-      url: https://github.com/swri-robotics-gbp/marti_common-release.git
-      version: 3.4.1-1
+      url: https://github.com/ros2-gbp/marti_common-release.git
+      version: 3.5.1-1
```

Comment:
This is not a correct version for Foxy. I only see 3.4.0 in the release repository instance for foxy: https://github.com/ros2-gbp/marti_common-release#marti_common-foxy---340-1 which matches the index: https://github.com/ros2-gbp/marti_common-release/blob/61b3536fc5c1920aa518ad6dfd1daef0eacd3bb2/tracks.yaml#L70

Reply:
I'm not sure what's going on. If I use the ```--pull-request-only``` flag, it works, but generates incorrect tags and versions. If I omit that command, it wants to release for v3.4.1, and use the ```swri-robotics-gbp``` repo. If I go through the process, I eventually get this error:

```
Release Repository Push URL:
  <url>
    (optional) Used when pushing to remote release repositories. This is only
    needed when the release uri which is in the rosdistro file is not writable.
    This is useful, for example, when a releaser would like to use a ssh url
    to push rather than a https:// url.
  :{none}
    This indicates that the default release url should be used.
  ['git@github.com:swri-robotics-gbp/marti_common-release.git']: https://github.com/ros2-gbp/marti_common-release.git
Saving 'foxy' track.
==> Setting release repository remote url to 'https://github.com/ros2-gbp/marti_common-release.git'
==> git remote set-url origin https://github.com/ros2-gbp/marti_common-release.git
==> Testing for push permission on release repository
==> git remote -v
origin  git@github.com:ros2-gbp/marti_common-release.git (fetch)
origin  git@github.com:ros2-gbp/marti_common-release.git (push)
==> git push --dry-run
To github.com:ros2-gbp/marti_common-release.git
 ! [rejected]            master -> master (fetch first)
error: failed to push some refs to 'github.com:ros2-gbp/marti_common-release.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
Cannot push to remote release repository.
Hint: If you just typed in your username/password and you have two-factor authentication,see:
  http://wiki.ros.org/bloom/Tutorials/GithubManualAuthorization
```

Reply:
I've opened https://github.com/ros2-gbp/marti_common-release/pull/3 which should address the problems here (and explains why they're occurring).

Reply:
Thanks, I merged the other PR and will rebloom this